views/static/home.handlebars

```diff
@@ -2,21 +2,32 @@
   <div class="container text-center">
     <div class="row">
       <div class="my-5 col-12 col-md-6">
-        <h1 class="text-blue">Matching Donors with Needy people in the Society</h1>
+        <h1 class="text-success">Helping each other has never been more convenient</h1>
         <div class="my-5 row text-center">
           <div class="col-12 col-md-6 text-center">
             <h2>For Donors</h2>
-            <p class="text-muted">Donation can be made easy by simply uploading an item to the page. Eveyone can donate
-              or
-              become a donor. </p>
-            <a href="/auth/signup?uType=Donor" class="btn btn-outline-secondary">Start Donating</a>
+            <p class="text-muted">
+              We are here to ensure you can help or ask for help,
+              in the most convenient way possible. Helping others
+              is always an emotionally rewarding experience, and
+              now we aim to make it convenient too.
+              <br>
+              Join Us, and help make the world a better place:
+            </p>
+            <a href="/auth/signup" class="btn btn-outline-secondary">Sign up to Start Helping</a>
           </div>
           <div class="col-12 col-md-6 text-center">
             <h2>For Recipient</h2>
-            <p class="text-muted">This platform is built for anyone who is in need. Are you in need? Time to shop for $0
+            <p class="text-muted">This platform is built for anyone in need.
+              Asking for help is always difficult, but not anymore.
+              <br>
+              No awkward conversations to be had, just join us and find
+              something that can add to your life, on Helping Hands.
+              <br>
+              Sign up as a recipient:
             </p>
             <br>
-            <a href="/auth/signup?uType=Recipient" class="btn btn-outline-success">Sign Up & Receive</a>
```

Comment:
@neilvarunnaidu Here, param is removed in the URL. Please check your changes and add back the missing code.